platform/src/common/components/Collocation/AddMonitor/Table/index.jsx

```diff
@@ -6,11 +6,15 @@ import SearchBar from './SearchBar';
 import Pagination from './Pagination';
 import Button from '../../../Button';
 import DataTable from './DataTable';
+import moment from 'moment';
```

Comment:
Use _date-fns_ or _day js_ (which are already installed) instead of moment. It is currently a legacy library and will is likely to cause challenges in the near future.
https://www.npmjs.com/package/moment

Reply:
ok Faith thanks that, i used that because that is what i found but am going to change

Reply:
Am done with changing the library now i used (date-fns)